packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -364,6 +362,116 @@ export function getMiniflareOptions(
 	};
 }
 
+export function getPreviewMiniflareOptions(
+	normalizedPluginConfig: NormalizedPluginConfig,
+	viteConfig: vite.ResolvedConfig,
+): MiniflareOptions {
+	const entryWorkerConfig = normalizedPluginConfig.entryWorkerName
+		? normalizedPluginConfig.workers[normalizedPluginConfig.entryWorkerName]
+		: undefined;
+
+	const assetsDirectory = path.resolve(
+		viteConfig.root,
+		viteConfig.build.outDir,
+		'client',
+	);
+	const hasAssets = fs.existsSync(assetsDirectory);
+	const assets = hasAssets
+		? {
+				assets: {
+					routingConfig: {
+						has_user_worker: entryWorkerConfig ? true : false,
+					},
+					assetConfig: {
+						...(normalizedPluginConfig.assets.htmlHandling
+							? { html_handling: normalizedPluginConfig.assets.htmlHandling }
+							: {}),
+						...(normalizedPluginConfig.assets.notFoundHandling
+							? {
+									not_found_handling:
+										normalizedPluginConfig.assets.notFoundHandling,
+								}
+							: {}),
+					},
+					directory: assetsDirectory,
+					...(entryWorkerConfig?.assetsBinding
+						? { binding: entryWorkerConfig.assetsBinding }
+						: {}),
+				},
+			}
+		: {};
+
+	const workers: Array<WorkerOptions> = [
+		...(entryWorkerConfig
+			? [
+					{
+						...entryWorkerConfig.workerOptions,
+						...assets,
+						modules: [
+							{
+								type: 'ESModule',
+								path: path.resolve(
+									viteConfig.root,
+									viteConfig.build.outDir,
+									entryWorkerConfig.workerOptions.name,
+									'index.js',
+								),
+							} as const,
+						],
+					},
+				]
+			: [
+					{
+						...assets,
+						name: 'assets-only',
+						modules: [
+							{
+								type: 'ESModule',
+								path: '__dummy-module__',
+								contents: '',
+							} as const,
+						],
+					},
```

Comment:
I think something's wrong here 😕 

if I am understanding things correctly this is for assets-only projects right?

my issue is that if I comment this entire object out the build tests still pass 😕 

the `react-spa` ones should fail though, shouldn't they?

Reply:
(in either case I think that all the code we add to the package, if removed should cause at least one test to fail, otherwise something's not right in my opinion (we either don't cover features or like now something is just off))

Reply:
I just tried commenting this object out and the react-spa and static-mpa tests do fail. What are the steps to recreate this? Are you sure you rebuilt the plugin after the change?

Reply:
no I just re-tried and it does seem to trigger some failures... I was quite sure that it didn't 😕  (I was building the plugin in watch mode, I wonder if that could have caused my confusion)

sorry, it's all good, my bad! 🙇